package.json

```diff
@@ -9,7 +9,20 @@
     "preview": "vite preview",
     "type-check": "vue-tsc --noEmit",
     "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore",
-    "format": "prettier --write src/"
+    "format": "prettier --write src/ locales/",
```

Comment:
this will ensure that if deepl-translate-github-action runs to make a commit, it will prettify the locale json generated